static/.htaccess

```
@@ -6,6 +6,16 @@ RewriteRule (.*) https://%{SERVER_NAME}/$1 [R=301,L]
 # Disable directory listing and multi views
 Options -Indexes -MultiViews
 
+# Renaming Guides
+RewriteRule ^docs/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
+RewriteRule ^docs/guides/migrating-from-camunda-platform-7$ /docs/guides/migrating-from-camunda-7 [R=301,L]
+RewriteRule ^docs/next/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
+RewriteRule ^docs/next/guides/migrating-from-camunda-platform-7$ /docs/guides/migrating-from-camunda-7 [R=301,L]
+RewriteRule ^docs/8.2/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
+RewriteRule ^docs/8.2/guides/migrating-from-camunda-platform-7$ /docs/guides/migrating-from-camunda-7 [R=301,L]
+RewriteRule ^docs/8.1/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
+RewriteRule ^docs/8.1/guides/migrating-from-camunda-platform-7$ /docs/guides/migrating-from-camunda-7 [R=301,L]
```

```suggestion
RewriteRule ^docs/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
RewriteRule ^docs/next/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
RewriteRule ^docs/8.2/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
RewriteRule ^docs/8.1/guides/migrating-from-camunda-platform-7(.*)$ /docs/guides/migrating-from-camunda-7$1 [R=301,L]
```

I think these rules can be cut in half -- I don't think the second rule in each set will ever be hit, because the only URL it would match would already be caught by the previous rule.

Example: the second rule is intended to match the URL `docs/guides/migrating-from-camunda-platform-7` exactly. But the first rule (`^docs/guides/migrating-from-camunda-platform-7(.*)$`) would match that URL, and rewrite it as `/docs/guides/migrating-from-camunda-7`. 

If I'm wrong, and through testing you confirmed that the second rule in each set is necessary, then disregard!